nuxeo-distribution/nuxeo-server-hotreload-tests/src/test/java/org/nuxeo/ftest/server/hotreload/ITDevHotReloadTest.java

```diff
@@ -72,6 +74,24 @@ public void testHotReloadVocabulary() {
         assertEquals("child2", properties.get("label"));
     }
 
+    @Test
+    public void testHotReloadSequence() {
+       String storageConf = StorageConfiguration.defaultSystemProperty(StorageConfiguration.CORE_PROPERTY, StorageConfiguration.DEFAULT_CORE);
+       assumeTrue(StorageConfiguration.CORE_VCS.equals(storageConf));
+       assertEquals(1, getSequence());
+       assertEquals(2, getSequence());
+       assertEquals(3, getSequence());
+    }
+
+    protected int getSequence() {
+        // operation to get a sequence from hibernateSequencer
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put("sequenceName", "hibernateSequencer");
+        String response = (String) RestHelper.operation("javascript.getSequence", parameters);
+        Map result = RestHelper.CLIENT.getConverterFactory().readJSON(response, HashMap.class);
+        return Integer.parseInt((String) result.get("value"));
```

Comment:
Following comments on `RestHelper`, you don't need to get the value and check it. In fact only call is necessary to test the hot reload behavior here. It's the sequencer responsibility to perform this test. Here you should just test that you can perform the call, as you don't change anything about the sequencer data.

Reply:
So we call an operation and assume its successful because it doesn't error rather than checking the return type.

Reply:
Yes exactly. I agree it's not perfect, but I don't have solutions in mind to get the result without relying on nuxeo-java-client objects. We could probably do something, for instance serialize the response to String and return it. But I think it's a lot of complexity just to check if component is still reachable. Wdyt ?

Reply:
Let me check again that the test fails if I revert my fix, then I think its ok to go with this approach.

Reply:
I get a nice big error when I revert my fix so the test is fine.